src/FormHelpers.js

```diff
@@ -61,23 +61,19 @@ export const convertAPIError = (httpStatus, httpResponse) => {
  * Retrieve existing or default value from account.billingPreferences object
  *
  */
-export const getDefaultValue = (account, billingProp, defaultValue) => {
-  return !!account === true
-    && !!account.billingPreferences === true
-    && !!account.billingPreferences[billingProp] === true
-    ? account.billingPreferences[billingProp]
+export const getDefaultValue = (model, billingProp, defaultValue) => {
+  return !!model === true && !!model[billingProp] === true
+    ? model[billingProp]
     : defaultValue
 }
 
-export const getDefaultCardExpDate = (account) => {
-  if (!!account === false || !!account.billingPreferences === false) {
+export const getDefaultCardExpDate = (instrument) => {
+  if (!!instrument === false) {
     return ""
   }
 
-  return !!account.billingPreferences.cardExpdate.month
-  || !!account.billingPreferences.cardExpdate.year === true
-  ? account.billingPreferences.cardExpdate.month
-    + '/' +
-    account.billingPreferences.cardExpdate.year
-  : ""
+  return (
+    `${!!instrument.cardExpdate && !!instrument.cardExpdate.month ? instrument.cardExpdate.month : "" }` +
```

Comment:
1. Please put expressions on multiple lines.
2. When using `!!expr` its already confusing as is, please include `=== true` or `=== false`
3. Maybe create an accessor method `instrument.cardExpdateMonth` or `instrument.getExpdateMonth()` to hide the complexity.